silk/migrations/0009_explicit_id_fields.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations, models
+
```

Comment:
when remvoe the explicit per moderl ID, and reintroduce appconfig, remove this file and regen a new migration and commit on this PR please.

Reply:
Migration would not be required in appconfig based approach.

Reply:
that would be ok